src/routes/docs/quick-starts/apple/+page.markdoc

```diff
@@ -137,64 +106,71 @@ class Appwrite {
         )
     }
     
-    public func onLogout() async throws {
+    func onLogout() async throws {
         _ = try await account.deleteSession(
             sessionId: "current"
         )
     }
     
+    func getSession() async throws -> Session {
+        try await account.getSession(
+            sessionId: "current"
+        )
+    }
 }
-
 ```
 {% /section %}
+
 {% section #step-5 step=5 title="Create a login page" %}
 Add the following code to `ContentView.swift`.
 
 ```swift
 import SwiftUI
 
-class ViewModel: ObservableObject {
-    @Published var email: String = ""
-    @Published var password: String = ""
-}
-
 struct ContentView: View {
-    @ObservedObject var viewModel = ViewModel()
+    @State var email: String = ""
+    @State var password: String = ""
+    @State var session: Session?
+    
+    let appwrite = Appwrite()
```

Comment:
Let's keep the `ViewModel`, but move the `Appwrite` instance into it. We can also add proxy methods for `onRegister` and `onLogin` in the view model to call the appwrite functions

Reply:
I can go either way on `ViewModel` but I am recommending that we remove it since view models are a debatable topic in the SwiftUI community. I also feel like it adds unnecessary complexity to the docs and would be better suited for tutorials instead. What do you think?

Reply:
It's good practice to separate the data from the view in any framework, to promote modularity and improve testability. I agree it adds a bit of complexity but we need to strike a balance between simplicity and best practice. In this case, I think it's better to leave it. 

Although we could name it better like `LoginCredentials` or similar instead of just `ViewModel` to make it easier to reason about